lms/static/sass/course/_tabs.scss

```diff
@@ -9,4 +9,8 @@ div.static_tab_wrapper {
   section {
     margin: 0px 0px 20px;
   }
+
+  @media print {
```

Comment:
See screenshot below:
![screen shot 2015-04-17 at 4 00 04 pm](https://cloud.githubusercontent.com/assets/6232546/7209900/fff57b1c-e51a-11e4-93e5-e7afa08701c3.png)
It's the same border for every other border you've asked for a screenshot for, just that it exists in different parts of the CSS. (Note: this comment was previously on a previous line, that got eaten when it became an old diff)